packages/dropdown/src/Dropdown.ts

```diff
@@ -59,9 +62,16 @@ export class DropdownBase extends Focusable {
         target: HTMLElement,
         interaction: TriggerInteractions,
         content: HTMLElement,
+        menuContent: HTMLElement,
```

Comment:
Here is my reason for this override (you can talk me out of it if you like).

As a consumer of an API, such as `sp-action-menu` I felt that I would expect the `sp-open` event to report to me on the opening of the item I have put in the slot. I don't think I would expect it to report a component that is an internal implementation artefact of the `sp-action-menu`. In that case, the `sp-action-menu` constructs an `sp-popover` to contain the contents that I have passed in.

This is a minor detail, but worth making an early decision on. I am not dead set on this, I'm just going with my gut feel.

Reply:
This differentiation does seem useful, I'm still a little unclear on how the `content` value would be leveraged, however. Can you share some context on what you do with this value, as opposed to simply knowing that open/closed has happened? Maybe a test or example of what can/should be done with that additional data when it's received...

Reply:
A good question. My main reasoning for including it is because of the way overlays work. They reparent the content. So, unless you cache a reference to the content, you will have trouble finding via the DOM it after the overlay has opened because it will have been reparented into an `active-overlay`. This isn't a problem for my current use case, so we could remove the `content` part until it is requested.

Reply:
If you're not leveraging this API/data, and you felt that we could get by without it for a while, I'd be quite happy moving forward without it for now. Having less code to document/maintain/test feels like the right call at this moment. I worry there likely to be some additional API surface for overlay work soon, anyways, and we could benefit with a little more systematic approach to what opens/closes/etc at that time.